service/history/configs/config.go

```diff
@@ -345,6 +349,10 @@ func NewConfig(dc *dynamicconfig.Collection, numberOfShards int32, isAdvancedVis
 		QueuePendingTaskMaxCount:         dc.GetIntProperty(dynamicconfig.QueuePendingTaskMaxCount, 10000),
 		QueueMaxReaderCount:              dc.GetIntProperty(dynamicconfig.QueueMaxReaderCount, 2),
 
+		TaskSchedulerEnableRateLimiter: dc.GetBoolProperty(dynamicconfig.TaskSchedulerEnableRateLimiter, false),
```

Comment:
is it worth to add configs per queue?

Reply:
my main concerns is when rate limiter is enabled for some queue but not others, the rps should also be updated accordingly, which might be confusing. Since the rate limiter is designed be to a singleton that use by all the queues, I think having one feature flag should be enough.

The only benefit I can see is due to no-op tasks in the old transfer/timer queue implementation, the rate limiter should not be enabled, but can be enabled for other queues. But I think in 1.19 there's no blocker for enabling multi-cursor, so this should be not an issue.